README.md

```diff
@@ -18,10 +18,20 @@ python2.7 powergraph.py --host="server address" --port="server port" --user="all
 ```
 You can use the optional parameter ```--store``` in order to save
 the infos as json on tinydb. Without this parameter, the script will
-print on the terminal.
+print on the terminal. Besides, you can use ```--feedback``` with store
+in order to see the measures status.
 
 Run ```csvcreator.py``` like this:
 
 ```
 python2.7 csvcreator.py --jsonfile="generated_json_name"
```

Comment:
we can remove python 2.7 from the command specification. Given we added it at the top of file. In addition, ensure you chmod +x you Python script before pushing it.